### DISCUSSION TITLE
Enfugue GPT helper bot

### DISCUSSION THREAD

**Discussion by pxlmover** [painebenjamin/app.enfugue.ai#100] [Ideas]
I've created a GPT and have given it all of the information on the releases and quick start pages here on github.  This is a brilliant UI and I can't wait to see further improvements and additions.  I present to you the Enfuguerator! Give it a try! https://chat.openai.com/g/g-v07WmLite-enfuguerator

**Comment by painebenjamin** [comment]
This is an AWESOME idea. Thank you so much for putting this together - I'm definitely going to play around with it and see how accurate it is, but this seems like a great stopgap solution until I can make more learning material.

Thank you again, this is incredible!
